src/Cli.elm

```diff
@@ -140,6 +143,8 @@ options:
                                      instead of failing use one of: `yes`, `y`, `true`.
 
   --overrides                        Load an additional file to override parts of the original Open API file.
+
+  --dump-merged-to                   Dump the merged Open API spec to the given file.
```

Comment:
Can we change "dump" to "write"? I feel like "dump" could be misinterpreted in some cultures/interpretations to me "discard"